torch/autograd/__init__.py

```diff
@@ -212,8 +212,9 @@ def grad(
     retain_graph: Optional[bool] = None,
     create_graph: bool = False,
     only_inputs: bool = True,
-    allow_unused: bool = False,
-    is_grads_batched: bool = False
+    allow_unused: Optional[bool] = None,
+    is_grads_batched: bool = False,
+    materialized_grad: bool = False,
```

Comment:
materialized_grad -> materialize_grad (to follow the other arguments, we want verbs here)